python/experimental/compare_histograms.py

```diff
@@ -29,7 +28,7 @@ def compare_histograms(hist1, hist2, use_ks_test=False, significance_threshold=0
             # Perform the Kolmogorov-Smirnov test
             ks_statistic, p_value = stats.ks_2samp(hist1.values(), hist2.values())
             print(f"KS Statistic: {ks_statistic}, p-value: {p_value}")
-            if p_value < significance_threshold:
+            if p_value.all() < significance_threshold:
```

Comment:
I'm a bit surprised by this change. Did the return signature of `ks_2samp` change?

Reply:
Ah, I see. I only tested this script using 1-D histograms.